src/viewsProto2/StartLearn/StartLearn.js

```diff
@@ -56,8 +56,8 @@ function StartLearn() {
   );
   const [sel_lang_text, set_sel_lang_text] = useState(
     localStorage.getItem('apphomelang')
-      ? localStorage.getItem('apphomelang') === 'ta'
-        ? 'Tamil'
+      ? localStorage.getItem('apphomelang') === 'hi'
```

Comment:
please check it has impact on our current task

Reply:
Not Using anywhere